g2p_seq2seq/g2p.py

```diff
@@ -90,26 +95,23 @@ def __init__(self, model_dir):
     print("Creating %d layers of %d units." % (num_layers, size))
     self.model = seq2seq_model.Seq2SeqModel(len(self.gr_vocab),
                                             len(self.ph_vocab), self._BUCKETS,
-                                            size, num_layers, 0, batch_size,
-                                            0, 0, forward_only=True)
-    self.model.saver = tf.train.Saver(tf.all_variables(), max_to_keep=1)
+                                            size, num_layers,
+                                            self.params.max_gradient_norm,
+                                            self.params.batch_size,
+                                            self.params.learning_rate,
+                                            self.params.lr_decay_factor,
+                                            forward_only=self.params.decode)
+    self.model.saver = tf.train.Saver(tf.all_variables(), max_to_keep=1,
+                                      write_version=saver_pb2.SaverDef.V1)
```

Comment:
This part does not seem to be related to a current change, it must be a change from previous issue fix.